### PR TITLE
history and singlevideo page

### PR DESCRIPTION
**Pull request by preethasri**
added history page
added singlevideo page
Screenshot (34)

Screenshot (35)

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]
Looks good to merge 👍🏻

File: src/Context/HistoryContext.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+    catch(err){
+        console.log(err)
+    }
+}
+const removeFromHistoryVideos=async video=>{
+    try{
+        const response=await removeHistory(token,video)
+        setHistoryVideos(response.data.history)
+        toast("video removed fro history")
```
Comment (gopalbbharadva) [inline src/Context/HistoryContext.js:49]: typo  of "from"

**Review by gopalbbharadva — APPROVED** [review]
Feature working fine , good to go.

File: src/Context/HistoryContext.js (hunk 1 above)
Comment (preethasri) [reply src/Context/HistoryContext.js:49]: yeah will change that

**Pull request merged**